### PR TITLE
Create stale workflow

### PR DESCRIPTION
**Pull request by Mdnaimulislam**
Fixes https://github.com/pykale/pykale/projects/10#card-84917959

### Description
Stale workflow has been created to mark stale issues and pull requests.

### Status
**Ready**

### Types of changes
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] In-line docstrings updated.
- [ ] [Source for documentation at docs](https://github.com/pykale/pykale/tree/main/docs/source) manually updated for new API.

### CODE REVIEW

**Comment by Mdnaimulislam** [comment]
pre-commit has been checked for all the files.

**Comment by Mdnaimulislam** [comment]
Docs build fails on Sphinx==5.2.0.post0. It's a recent bug on new version. They didn't fixed the bug yet. 
Possible solution: Use the previous version 5.1.1.

Source: https://github.com/readthedocs/sphinx_rtd_theme/issues/1343

**Comment by haipinglu** [comment]
> Docs build fails on Sphinx==5.2.0.post0. It's a recent bug on new version. They didn't fixed the bug yet. Possible solution: Use the previous version 5.1.1.
> 
> Source: [readthedocs/sphinx_rtd_theme#1343](https://github.com/readthedocs/sphinx_rtd_theme/issues/1343)

The other PR has no issue with docs, see and compare with https://readthedocs.org/projects/pykale/builds/18167003/ to understand deeper.

**Comment by Mdnaimulislam** [comment]
This is the error message beacuase of Sphinx v5.2.0.post0 on my PR yesterday.


This PR didn't had the problem beacuse the pending version v5.2.1 has been released 3 hours ago. I am confident that it will not create any problem now. Can you run it again now once?

**Comment by haipinglu** [comment]
@shuo-zhou Please think about setting the feasible/appropriate days in this workflow to discuss with me before merging.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies, run tests, and report the coverage with a variety of Python versions and OSs
```
Comment (haipinglu) [inline .github/workflows/stale.yml:1]: Wrong doc. Please fix

Comment (Mdnaimulislam) [reply .github/workflows/stale.yml]: I have fixed it. I will be careful about it next time. Thanks.

**Review by haipinglu — APPROVED** [review]

**Pull request merged**